scripts/generate-api-models.sh

```diff
@@ -40,7 +40,7 @@ declare -a apis=(
   "./definitions/cgn/merchants https://raw.githubusercontent.com/pagopa/io-backend/$IO_BACKEND_VERSION/api_cgn_operator_search.yaml"
   # PN APIs
   "./definitions/pn https://raw.githubusercontent.com/pagopa/io-backend/$IO_BACKEND_VERSION/api_pn.yaml"
-  "./definitions/pn/aar https://raw.githubusercontent.com/pagopa/io-messages/ed902158c99345dde8e1a37c78252d360e6d6c34/apps/send-func/openapi/aar-notification.yaml"
+  "./definitions/pn/aar https://raw.githubusercontent.com/pagopa/io-messages/082438e9f1ef0824559da4a99a96cd2233fe508c/apps/send-func/openapi/aar-notification.yaml"
```

Comment:
This should be updated to 
https://raw.githubusercontent.com/pagopa/io-messages/send-func@1.3.1/apps/send-func/openapi/aar-notification.yaml